.travis.yml

```diff
@@ -1,32 +1,52 @@
-language: perl
+language: minimal
+dist: bionic
 
-perl:
```

Comment:
not testing on different perl versions anymore ... a particular reason for this ?

Reply:
Older perls were failing in Travis with an error, like this one: https://travis-ci.com/ccremer/znapzend/jobs/285685206. I cannot decipher that.

Most likely the reason is a misconfiguration in travis (config).